cabal-install/tests/UnitTests/Distribution/Client/Dependency/Modular/DSL.hs

```diff
@@ -104,6 +107,7 @@ data ExampleDependency =
     -- | Dependency on a language version
   | ExLang Language
 
+type ExPreference = (String, ExampleVersionRange)
 
```

Comment:
I think that converting `ExPreference` into a data type would make the test cases easier to read.